config/initializers/apipie.rb

```diff
@@ -27,6 +27,8 @@
     trans
   end
   config.help_layout = 'apipie_dsl/apipie_dsls/help.html.erb'
+  config.default_model_description = ->(model) { N_("A class representing %s object") % model.model_name.singular.humanize(capitalize: false) }
```

Comment:
The use of `N_()` here makes me think it will never be translated. `N_()` is only a marker so I'd expect it to be either `_()` or `Apipie.translate()`. Probably the former, but I don't know if it'll work with scoping.

Reply:
@ekohl does this change https://github.com/theforeman/foreman/pull/9867/commits/d2b3052cd28b95ab3750fe0fd58c7277f3f795ea looks correct, also where i can see if this change is working in our documentation page, i can't find it.

Reply:
> The use of `N_()` here makes me think it will never be translated. `N_()` is only a marker so I'd expect it to be either `_()` or `Apipie.translate()`. Probably the former, but I don't know if it'll work with scoping.

If we do `_()` or `Apipie.translate()` here, it will be still evaluated during initialization time (class parsing). Although, that's my dumbass, I can fix probably fix this in apipie-dsl. Something like `if a block is passed, it will be called on docs demand, e.g. when to_docs is called on an object`.

Reply:
I don't think it is. You're dropping parameter descriptions on an API endpoint, which I think can't be determined by default.

Instead it should be updating the class descriptions for apipie-dsl. You can search for `apipie :class, desc: "A class representing` and drop `desc: ...` there. https://github.com/theforeman/foreman/pull/9771 has already highlighted those, but perhaps there are more.

If you have a running Foreman, you can go to `/templates_doc` and see the rendered result.